pages/ecosystem.vue

```diff
@@ -178,14 +185,22 @@ const membersByTier: MembersByTier = tiersNames.reduce((acc, tierName) => {
 
 const categoryFiltersAsString = computed(() => categoryFilters.value.join(","));
 
+const selectedTierDescription = computed(() => {
+  const tier = tiers.find((tier) => tier.name === selectedTab.value);
+  return tier?.description || "";
+});
+
```

Comment:
There code is repeated with this and `getTierDescription`. Now that I'm looking at them, I think that instead of calculating this value every time a tab is selected, we can create an object at the beginning with this structure:

```
{
  [tierName]: tierDescription
}
```

so we only have to access an object every time a tab is selected.

Reply:
Refactored in https://github.com/Qiskit/qiskit.org/pull/3322/commits/2f5afb609e7eeb40b48baee8467796dcd8a97fba